output/cloud/expv2/mapping.go

```diff
@@ -10,21 +11,37 @@ import (
 )
 
 // TODO: unit test
-func mapTimeSeriesLabelsProto(tags *metrics.TagSet) []*pbcloud.Label {
+func mapTimeSeriesLabelsProto(tags *metrics.TagSet) ([]*pbcloud.Label, map[string]struct{}) {
 	labels := make([]*pbcloud.Label, 0, ((*atlas.Node)(tags)).Len())
+	discardedLabels := make(map[string]struct{})
```

Comment:
Considering that this is expected to be empty most of the time, I'm for not allocating it and allocate only if we find at least one item.

Reply:
:heavy_check_mark: with the latest commit